vllm_spyre/model_executor/model_loader/spyre.py

```
@@ -272,11 +289,23 @@ def __init__(
             print(f"[SpyreCausalLM] model type {self.config.model_type} "
                   f"not supported in ContinuousBatchingFmsModel")
 
```

Should we raise an Error here?